### PR TITLE
added nav subtitle

### PR DESCRIPTION
**Pull request by paulhan221**

### CODE REVIEW

File: public/assets/custom-neue.css [hunk 1]
```diff
@@ -3,6 +3,10 @@
 .navigation {
     background-color: #4e2b63;
 }
+
+.navigation__subtitle {
+  color: white;
```
Comment (DFurnes) [inline public/assets/custom-neue.css]: This shouldn't be necessary to override.

File: public/assets/custom-neue.css [hunk 2]
```diff
@@ -3,6 +3,10 @@
 .navigation {
```
Comment (DFurnes) [inline public/assets/custom-neue.css]: You can use the [header](http://neue.dosomething.org/#header) pattern (alongside the navigation pattern's floating modifier, which you're already using) to do this without an override.

File: public/assets/custom-neue.css (hunk 1 above)
Comment (paulhan221) [inline public/assets/custom-neue.css]: it shows up as the same color as the background otherwise
it seems to be the default color unless specified

**Comment by angaither** [comment]
:+1:

**Comment by DFurnes** [comment]
:shipit:!

**Pull request merged**